hotel-api-services/src/main/java/com/fponce/hotelapp/hotel/service/HotelService.java

```diff
@@ -0,0 +1,10 @@
+package com.fponce.hotelapp.hotel.service;
```

Comment:
Why is the package structure different from the others?, Shouldn't be any standard?

Reply:
Yes, it should be standard. I changed it to have the same package base.